operations/System_Recovery/PBS_Service_Recovery.md

```diff
@@ -10,9 +10,8 @@ The following covers restoring PBS data.
 
 ## Service recovery for PBS
 
-To restore PBS Professional data from backup, follow section
-*10.7.7 Restore PBS home directory from backup* in the
-**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+To restore Portable Batch System data from backup, see *Restore PBS home directory from backup* in the
+**HPE Portable Batch System Installation Guide: CSM on HPE Cray EX Systems (S-8055)**.
```

Comment:
```suggestion
To restore Portable Batch System data from a backup, see *Restore PBS home directory from a backup* in the
**HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray EX Systems (S-8063)**.
```

Reply:
Added a few minor changes from the CSM PRs I'm working on. Also, for the CSM 1.6 release, the current plan is to move all the WLM documentation into the USS guides. I've updated this PR to reflect that.

Reply:
fixed